content/blog/2021/10/05/OptimizeYourPython.adoc

```diff
@@ -0,0 +1,438 @@
+----
+
+For the `SolverConfig` above, we use the default strategies, use the model we defined above, and set it terminate after 30 seconds.
+
+Finally, we pass the `SolverConfig` and the
+problem instance to the `solve` function, which return the last best solution found:
+
+[source,python]
+----
+from optapy import solve
+
+solution = solve(solverConfig, generate_problem())
+print(solution)
+----
+
+The solution returned is a `TimeTable` instance
+of the best solution found.
+When the solution is formatted into a table, it should look similar to this:
+[source,text]
+----
+|------------|------------|------------|------------|
+|            | Room A     | Room B     | Room C     |
+|------------|------------|------------|------------|
+| MON 08:30: |            | Math       | History    |
+|            |            | A. Turing  | I. Jones   |
+|            |            | 9th grade  | 10th grade |
+|------------|------------|------------|------------|
+| MON 09:30: |            | Math       | History    |
+|            |            | A. Turing  | I. Jones   |
+|            |            | 10th grade | 9th grade  |
+|------------|------------|------------|------------|
+| MON 10:30: |            | Math       | English    |
+|            |            | A. Turing  | I. Jones   |
+|            |            | 10th grade | 9th grade  |
+|------------|------------|------------|------------|
+| MON 13:30: | Math       | Spanish    |            |
+|            | A. Turing  | P. Cruz    |            |
+|            | 10th grade | 9th grade  |            |
+|------------|------------|------------|------------|
+| MON 14:30: | Math       | English    |            |
+|            | A. Turing  | P. Cruz    |            |
+|            | 9th grade  | 10th grade |            |
+|------------|------------|------------|------------|
+| TUE 08:30: | Physics    | Spanish    |            |
+|            | M. Curie   | P. Cruz    |            |
+|            | 9th grade  | 10th grade |            |
+|------------|------------|------------|------------|
+| TUE 09:30: | Chemistry  |            | English    |
+|            | M. Curie   |            | I. Jones   |
+|            | 10th grade |            | 9th grade  |
+|------------|------------|------------|------------|
+| TUE 10:30: | Physics    | Spanish    |            |
+|            | M. Curie   | P. Cruz    |            |
+|            | 10th grade | 9th grade  |            |
+|------------|------------|------------|------------|
+| TUE 13:30: | French     |            | Biology    |
+|            | M. Curie   |            | C. Darwin  |
+|            | 10th grade |            | 9th grade  |
+|------------|------------|------------|------------|
+| TUE 14:30: | Chemistry  | Geography  |            |
+|            | M. Curie   | C. Darwin  |            |
+|            | 9th grade  | 10th grade |            |
+|------------|------------|------------|------------|
+----
```

Comment:
I think this output needs to be at the top, to **show** what we're building before we start building it.

Alternatively - even better I think - would be to Inkscape edit the svg next to this png:
https://github.com/kiegroup/optaplanner/blob/main/optaplanner-docs/src/modules/ROOT/images/quickstart/school-timetabling/schoolTimetablingInputOutput.png
and replace the OptaPlanner icon with the words "OptaPy" and use that at the start of this blog post as well as in the optapy.org README. Then the console output can stay at the end.